superset-frontend/src/components/Menu/LanguagePicker.tsx

```diff
@@ -17,15 +17,17 @@
  * under the License.
  */
 import React from 'react';
-import PropTypes from 'prop-types';
 import { NavDropdown, MenuItem } from 'react-bootstrap';
 
-const propTypes = {
-  locale: PropTypes.string.isRequired,
-  languages: PropTypes.object.isRequired,
-};
+export interface LanguagePickerInterface {
```

Comment:
In general, we should only export interfaces that are actually required in other files. Let's remove `export` from anything that's not required elsewhere

Reply:
done. removed unnecessary exports